app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
```

Comment:
Thanks for pulling this out of the UsersController. This makes a lot more sense to be in a separate controller than all of the functionality in one.